src/sass/components/_org-cards.scss

```diff
@@ -1,4 +1,6 @@
 .has-nested-card {
+  // remove extra space on org action dropdown
+  --sds-c-card-body-spacing-inline-end: 0;
```

Comment:
This makes me nervous... isn't it an undocumented part of SLDS that could change?

Reply:
Darn, yes, it is in "beta" at the moment and I didn't catch that. I would be surprised if they don't go that way but we can update this here, and there is another place I am using them (in buttons.scss I think).